src/Umbraco.Cms.ManagementApi/Controllers/ExamineManagement/HasIndexRebuiltExamineManagementController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Examine;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.ManagementApi.Factories;
+using Umbraco.Cms.ManagementApi.ViewModels.ExamineManagement;
+
+namespace Umbraco.Cms.ManagementApi.Controllers.ExamineManagement;
+
+[ApiVersion("1.0")]
+public class HasIndexRebuiltExamineManagementController : ExamineManagementControllerBase
```

Comment:
```suggestion
public class IndexExamineManagementController : ExamineManagementControllerBase
```

Rename controller to match new operation name